posts/2015-05-24-es6-gotchas.md

```diff
@@ -46,6 +46,21 @@ export default class ComponentName extends React.Component {
   }
 ~~~
 
+Another alternative syntax is to use the `::` ES7 [bind](https://github.com/zenparsing/es-function-bind)
+
+~~~js
+export default class ComponentName extends React.Component {
+
+  handleClick(e) => {
+    ...
+  }
+  
+  render(){
+    return (
+      <button onClick={::this.handleClick}> click </button> 
```

Comment:
FWIW, I think the binary form of the `::` operator is much more powerful than the unary form. It gives you a beautiful, chainable way to invoke the equivalent of `func.call`, and cannot be expressed early so easily with a simple `func.bind(this)`.